libs/accounts/src/lib/transfer-container.tsx

```diff
@@ -44,7 +45,7 @@ export const TransferContainer = ({ assetId }: { assetId?: string }) => {
         symbol: account.asset.symbol,
         name: account.asset.name,
         decimals: account.asset.decimals,
-        balance: addDecimal(account.balance, account.asset.decimals),
+        balance: toBigNum(account.balance, account.asset.decimals),
```

Comment:
I'm not a fan of using BigNum. Reason why is that they are harder to compare when used as memo param.
`new BigNumber(0) !== new BigNumber(0)`